packages/vue/src/components/molecules/SfComponentSelect/SfComponentSelect.vue

```diff
@@ -52,7 +52,7 @@
           >
             <slot />
           </ul>
-          <slot name="cancel">
+          <slot name="cancel" v-bind="{ cancelLabel, cancelHandler }">
```

Comment:
`cancelHandler` should be changed to `closeHandler` or the otherwise the rest should be changed to 'cancelHandler`, I don't know the proper name here :)

Reply:
nice catch :)